### PR TITLE
Material UI fix

### PR DESCRIPTION
**Pull request by basvo**
Hello Alan, thanks for developing this, it saved me big time restoring 100+ transactions in YNAB. I needed to change the way MaterialUI is initialized. This is because of a fix in Material UI? https://github.com/mui-org/material-ui/issues/12387. Not an expert with JS so please check if this fix is correct.

### CODE REVIEW

**Comment by bradymholt** [comment]
@alanrussian - This does seem to fix this error that is currently happening:

```
Uncaught TypeError: Cannot destructure property 'Button' of 'window.material-ui' as it is undefined.
    at main.js:9
    at main.js:428
```

Are you up for merging this and updating the CDN version?  Let me know if I can do anything to help.  Thanks!

File: README.md [hunk 1]
```diff
@@ -9,3 +9,13 @@ This uses the YNAB API to read your transactions. It presents you with your tran
 - The new transactions that get created will need to be approved and are flagged with the color red.
 
 - Since the old, deleted transactions aren't themselves resurrected, they'll continue to show up in the app if you refresh it.
+
+## Hosted version
+
+https://basvo.github.io/ynab-transaction-resurrector/
```
Comment (alanrussian) [inline README.md:15]: Does this go to a fork of the project? Can we have it go to this project?

File: README.md [hunk 2]
```diff
@@ -9,3 +9,13 @@ This uses the YNAB API to read your transactions. It presents you with your tran
...
+
+## Credits
+
+Original credits to Alan Russian (https://github.com/alanrussian/ynab-transaction-resurrector)
+
+I just fixed a small issue with Material UI and ran into the memo max length issue when restoring 100+ transactions I accidentally removed from YNAB.
```
Comment (alanrussian) [inline README.md:21]: Feel free to replace "I" with your name if you'd like!

File: main.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 (function () {
-  const clientId = '60b252599fc55fd7f79c91370e37e141408cf38df7e0fa129130afd960bb683e';
+  const clientId = '07592e512632cbce9dc6a8ff5064296c78ef1c2ba8b88b2004d1429ed6fbffed';
```
Comment (alanrussian) [inline main.js:4]: What's the reason for changing the client ID? Does this need to match a URL set up with YNAB? I can update my ID to support the GitHub.io url if desired. I think it's currently set up for https://cdn.rawgit.com/alanrussian/ynab-transaction-resurrector/69f7f489/index.html

File: main.js [hunk 4]
```diff
@@ -176,7 +176,7 @@
       const transactions = this.props.data.transactions.filter(transaction => transaction.deleted);
       const result = [];
       for (var i = 0; i < transactions.length; i++) {
-        const transaction = transactions[i];
+        var transaction = transactions[i];
```
Comment (alanrussian) [inline main.js:179]: Nit: could we keep transaction const and create a mutable memo variable like we did with payeeName?

**Review by alanrussian — COMMENTED** [review]
Hey thanks for submitting this pull request! I'm sorry I missed it.

**Comment by alanrussian** [comment]
Thanks for the ping, @bradymholt!

**Comment by bradymholt** [comment]
@alanrussian - It looks like @basvo 's fork updated some other things so he could use it separately.  I just submitted https://github.com/alanrussian/ynab-transaction-resurrector/pull/2, which should only make the necessary changes to fix the problem.  So, maybe you could review/merge https://github.com/alanrussian/ynab-transaction-resurrector/pull/2 and close this one.  TIA.

**Comment by alanrussian** [comment]
Closing this since we merged #2.

**Comment by basvo** [comment]
> 
> 
> @alanrussian - It looks like @basvo 's fork updated some other things so he could use it separately. I just submitted #2, which should only make the necessary changes to fix the problem. So, maybe you could review/merge #2 and close this one. TIA.

Yes correct, needed to update the YNAB API key among other things to make it work. Thanks for creating a separate PR and closing this one.

**Pull request closed without merging**